src/types.js

```diff
@@ -382,3 +382,41 @@ export type Team = {
   updated_at: string,
   workflow: Workflow,
 };
+
+/* Iterations */
+
+export type IterationStats = {
+  average_cycle_time: number,
+  average_lead_time: number,
+  num_points: number,
+  num_points_done: number,
+  num_points_started: number,
+  num_points_unstarted: number,
+  num_stories_done: number,
+  num_stories_started: number,
+  num_stories_unestimated: number,
+  num_stories_unstarted: number,
+};
+
+export type Iteration = {
+  created_at: string,
+  updated_at: string,
+  start_date: string,
+  end_date: string,
+  description: string,
+  id: ID,
+  name: string,
+  entity_type: string,
+  status: string,
```

Comment:
This can be a union-type:

`type IterationStatus = 'unstarted' | 'started' | 'done';`